apps/native/app/src/screens/applications/applications.tsx

```diff
@@ -71,84 +56,75 @@ const { useNavigationOptions, getNavigationOptions } =
     },
   )
 
+interface SortedApplication {
+  incomplete: Application[]
+  inProgress: Application[]
+  completed: Application[]
+}
+
+export const sortApplicationsStatus = (
+  applications: Application[],
+): SortedApplication => {
+  const incomplete: Application[] = []
+  const inProgress: Application[] = []
+  const completed: Application[] = []
+
+  applications.forEach((application) => {
+    if (
+      application.state === ApplicationResponseDtoStatusEnum.Draft ||
+      application.state === 'prerequisites'
```

Comment:
As I mentioned above, this sorting is taken from the service portal, would advice to check if it is correct and still relevant 😄